modules/seqexec/server/src/main/scala/seqexec/server/gems/Gems.scala

```diff
@@ -63,33 +118,33 @@ object Gems {
     def isActive[T: DetectorStateOps: Eq](v: T): Boolean = v === DetectorStateOps[T].trueVal
   }
 
-  sealed trait Ngs1DetectorState extends Product with Serializable
-  object Ngs1DetectorState {
-    case object On extends Ngs1DetectorState
-    case object Off extends Ngs1DetectorState
+  sealed trait Cwfs1DetectorState extends Product with Serializable
+  object Cwfs1DetectorState {
+    case object On extends Cwfs1DetectorState
+    case object Off extends Cwfs1DetectorState
 
-    implicit val ngs1DetectorStateEq: Eq[Ngs1DetectorState] = Eq.fromUniversalEquals
-    implicit val ngs1DetectorStateDetectorStateOps: DetectorStateOps[Ngs1DetectorState] =
+    implicit val cwfs1DetectorStateEq: Eq[Cwfs1DetectorState] = Eq.fromUniversalEquals
```

Comment:
We have been converting some of these to `Enumerated` like https://github.com/gemini-hlsw/ocs3/blob/develop/modules/seqexec/model/src/main/scala/seqexec/model/enum/FPUMode.scala

It would give you an `Eq` and `Arb` automatically